### PR TITLE
Update outdated cleanup jobs

### PR DESCRIPTION
**Pull request by Sunnatillo**
This PR updates outdated cleanup jobs: 

-  Rename files properly
-  Add step to cleaunup private cloud
- Update cleanup function
- changes the jenkins worker

### CODE REVIEW

**Review by smoshiur1237 — COMMENTED** [review]
/lgtm

File: jenkins/scripts/clean_resources.sh [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        
+        # Delete the server
+        openstack server delete "${server_id}"
+        fi
+    fi
+    done
+}
+
+# Install openstack client
+pip install python-openstackclient==7.0.0
```
Comment (tuminoid) [inline jenkins/scripts/clean_resources.sh:48]: make this version a variable

```suggestion
pip install python-openstackclient=="${CLIENT_VERSION}"
```
and add CLIENT_VERSION=7.0.0 on top of the file

**Review by tuminoid — COMMENTED** [review]
nits

**Comment by Sunnatillo** [comment]
> nits

addressed all of them

File: jenkins/scripts/clean_resources.sh [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+            # Convert server creation date to seconds since the epoch
+            server_time=$(date --date="${created_at}" +%s)
+
+            # Calculate the age of the server
+            server_age=$((current_time - server_time))
+
+            # Check if the server is older than 6 hours
+            if [[ "${server_age}" -gt "${age_threshold}" ]]; then
+            echo "Deleting server: ${server_id} (Name: ${server_name}, Created at: ${created_at})"
```
Comment (tuminoid) [inline jenkins/scripts/clean_resources.sh:38]: indentation is still missing

**Review by tuminoid — COMMENTED** [review]
identation issue remains, otherwise lgtm

File: jenkins/scripts/clean_resources.sh (hunk 2 above)
Comment (Sunnatillo) [reply jenkins/scripts/clean_resources.sh]: done

**Review by tuminoid — COMMENTED** [review]
/approve

**Review by smoshiur1237 — COMMENTED** [review]
/lgtm

**Pull request merged**